Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java

```diff
@@ -3321,7 +3321,7 @@ public void shareTextMessage(String title, String message) {
                     sendIntent.setType("text/plain");
                    if (thumbnailBitmap != null &&  Build.VERSION.SDK_INT > 28) {
                        Uri thumbnailUri = getImageUri(context, thumbnailBitmap);
-                       ClipData clipData = ClipData.newUri(context.getContentResolver(), "ThumbnailImage", thumbnailUri);
+                       ClipData clipData = ClipData.newUri(context.getContentResolver(), "ThumbnailImage" + System.currentTimeMillis(), thumbnailUri);
```

Comment:
Did you test it sharing it in what'sapp?

Reply:
no need fixed in my PR